### DISCUSSION TITLE
suddenly there is a space before a number's unit indicator ("4.91 G")

### DISCUSSION THREAD

**Discussion by neeels** [brndnmtthws/conky#1309] [Q&A] [answered]
I'm using conky to produce the i3bar text in i3wm, have been for years.
It always looked like this:

` wlan0(v0B ^0B 84%)  ⇆3.00K ⌂4.91G √5.91G`
Since upgrading my OS at some point, also some while back, suddenly there are spaces before the unit indications:

`wlan0(v0 B ^0 B 84%) ⇆3.00 K ⌂4.91 G √5.91 G`

I find this quite annoying, now it looks all torn up, especially the wlan0 uplink downlink numbers.
is there a way to get rid of these spaces again?
A long time ago i had already looked in the conky source code trying to find where that space came from, no easy success.

while i'm at it, it would also be cool if i could fix the units of the wlan0 uplink downlink to always show in K, so i can always see at a glance whether it's a really big or really small number. When the units change, 1.2M and 1.2K look too similar, and i need more brain cycles to interpret the meaning.

Thanks!

**Comment by brndnmtthws** [comment, marked as answer]
Do you have [`units_spacer`](https://conky.cc/config_settings#units_spacer) set?

**Reply by neeels** [reply]
thanks for the hint, i tried to set it but i get

`conky: Unknown setting 'units_spacer'`

from

 > conky --version
 conky 1.12.2 compiled 2022-04-15 for Linux x86_64

I see now in git that units_spacer was added juuust after 1.12.2 was released, so my version doesn't have it yet.
After I built conky from source (`conky 1.16.2_pre`) it works as expected!
Thanks!

I guess there is no solution for the part of my question concerning making the units non-adaptive? i.e. always show network traffic in KB, instead of switching between B, KB, M
